docs(executor): first round of de-flowing (#5637)
jina-ai/jina: docs/concepts/executor/basics.md

(executor)=

# Basics

```{tip}
Executors use `docarray.DocumentArray` as input and output data structure. [Read DocArray's docs](https://docarray.jina.ai) to see how it works.
```

> An {class}`~jina.Executor` is a self-contained microservice with a gRPC gateway. 

Reviewer: ```suggestion
An {class}`~jina.Executor` is a self-contained microservice exposed using the gRPC protocol. 
```
